packages/mds-geography-service/repository/index.ts

```diff
@@ -141,6 +142,35 @@ class GeographyReadWriteRepository extends ReadWriteRepository {
     return []
   }
 
+  /**
+   * Fetches geographies by ids, returns null for ids that don't exist in the database. Note: order is important!
```

Comment:
Curious about this response choice. So if no geography is found for 5 ids, the response would be:
`[null, null, null, null, null]` Is this consistent on how we handle these type of query and response?
Is the ID not returned in the geography list for the client to know which IDs returned without having to preserve order?

Reply:
I guess this was what specifically was asked for. Nvm.